custom_dc/cloudsql_env.list

```diff
@@ -1,7 +1,10 @@
 FLASK_ENV=custom
 USE_CLOUDSQL=true
+ENABLE_MODEL=true
 
+DC_API_ROOT=https://api.datacommons.org
```

Comment:
no need to set DC_API_ROOT as it has a default value in build/web_compose/compose.sh

Reply:
I don't see a default value in [compose.sh](https://github.com/datacommonsorg/website/blob/master/build/web_compose/compose.sh).

Reply:
sorry, the DOCKERFILE https://github.com/datacommonsorg/website/blob/master/build/web_compose/Dockerfile#L101-L105

Reply:
Done.